cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -251,6 +251,7 @@ struct parquet_column_device_view : stats_column_desc {
   uint8_t const* nullability;  //!< Array of nullability of each nesting level. e.g. nullable[0] is
                                //!< nullability of parent_column. May be different from
                                //!< col.nullable() in case of chunked writing.
+  bool output_as_byte_array;   //!< Indicates this list column is being written as a byte array
```

Comment:
Does `physical_type` show if a column should be written as byte array?

Reply:
In this PR this is extraneous and we can simply use `physical_type`. I'm not sure if it is required for the next one, but I will investigate the possibility of removing this. Thanks, this is a good catch.

Reply:
Actually, this isn't correct. List tests started failing when I went off the physical type alone. I've added this back.